.github/workflows/MarkdownLinksCheck.yml

```diff
@@ -0,0 +1,19 @@
+name: Check Markdown links
+
+on: 
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
+    - uses: gaurav-nelson/github-action-markdown-link-check@2a60e0fe41b5361f446ccace6621a1a2a5c324cf
```

Comment:
Can't you use a tagged version instead? using a sha looks really weird

Reply:
using SHA is just more safe (e.g. the code in a tag could be - theoretically - replaced by the 3rd party vendor via removing the tag and creating the new tag with the same name and the new code)
Is actually what GitHub currently suggests for the 3rd-party actions
https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/security-hardening-for-github-actions#using-third-party-actions
But I agree, with SHA you have to dig into the action-repo to figure out which vesion is currently used. Hope GitHub will enable one day using a Tag **together** with SHA or something like that.

Reply:
I think this is overkill for us, specifically considering that everything is opened source already and there is absolutely no secret info to gain that is not visible. Using a tag would just be sufficient in my opinion. 
And if an action would in the very very very unlikely even delete our content..so what. We can add it back. So I would still suggest using tags